### PR TITLE
[ENH] Propose to use `*_channels.json` for the description of coordinate systems

### PR DESCRIPTION
**Pull request by JuliusWelzel**
Hello, 

as discussed [before](https://github.com/bids-standard/bids-specification/issues/1488), we would like to move the description of coordinate system information for motion data from the `*_motion.json` to the `*_channels.json` file.

@sjeung already implemented this in her [example datasets](https://github.com/sjeung/bids-examples/commit/acc012bea750c5b638fe67f8e2f6bb0327f1f2eb), I will not get around to do this before my holiday :(
We updated the schema accordingly.

Please check if the PR seems ok.

Best,
Julius

### CODE REVIEW

File: src/schema/rules/tabular_data/motion.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ motionChannels:
   columns:
     name__channels: required
     component: required
+    reference_frame__motion: recommended
```
Comment (JuliusWelzel) [inline src/schema/rules/tabular_data/motion.yaml:16]: I am not sure if this makes sense, or if it should be moved to the `*_channels.json` completly

**Review by sappelhoff — COMMENTED** [review]
we also need to remove the references to `RotationRule`, `RotationOrder`, and `SpatialAxes` here:

https://github.com/bids-standard/bids-specification/blob/8c9fa11e20d59f3ea64a3397b25cf9d45c09e3ef/src/schema/rules/sidecars/motion.yaml#L74-L75

Else they'll keep showing up as recommended fields in `motion.json`.

Instead we need to add some schema magic to make them show up under `channels.json` **IF** the modality is motion. cc @tsalo @effigies 

---

furthermore, I think the `reference_frame` column for `channels.tsv` should also be mentioned here:

https://github.com/bids-standard/bids-specification/blob/8c9fa11e20d59f3ea64a3397b25cf9d45c09e3ef/src/modality-specific-files/motion.md?plain=1#L140-L144

and link to the new section that you are introducing ("reference frame description")

File: src/schema/rules/tabular_data/motion.yaml (hunk 1 above)
Comment (sappelhoff) [reply src/schema/rules/tabular_data/motion.yaml:16]: no, this is needed here :+1:

**Comment by JuliusWelzel** [comment]
I implemented @sappelhoff comments in the previous commits. @sjeung can you please check if my suggestion [for introducing the `reference_frame` column](https://github.com/bids-standard/bids-specification/pull/1524/commits/179c4248206958a612f259b51bf0f21c5e0e326b) is sufficient for now?

**Comment by sjeung** [comment]
Hi! Both the example and the description in the specification look good from my side. 

@JuliusWelzel could you remind me if your example data set is up-to-date?
My updated examples can be found [here ](https://github.com/sjeung/bids-examples/tree/motion_029). 

For validator I only removed the fields from motion.json. 
 @tsalo @effigies @sappelhoff, Do you have any suggestions on whether to validate the channels.json according to [this part of the spec](https://github.com/sjeung/bids-validator/commit/c6810aa339456816ca6ff71e952a1990e71ecdb5)?

**Comment by effigies** [comment]
Started to comment here, refactored here: https://github.com/bids-standard/bids-specification/pull/1591

Diff from this PR: https://github.com/JuliusWelzel/bids-specification/compare/master...effigies:bids-specification:channels-json

**Comment by effigies** [comment]
> Do you have any suggestions on whether to validate the channels.json according to [this part of the spec](https://github.com/sjeung/bids-validator/commit/c6810aa339456816ca6ff71e952a1990e71ecdb5)?

I don't know how to validate that in the legacy validator. In the schema validator, you can't currently validate anything inside column descriptions. I'm not sure what the best path forward for that would be. (Flagging @rwblair, as we'll need to think about this.)

**Comment by JuliusWelzel** [comment]
@sappelhoff This PR should be merged into `main` before the release as it clarifies how to describe coordinate systems for motion data

**Comment by effigies** [comment]
@JuliusWelzel Could you review #1591? It contains these proposals:

https://github.com/bids-standard/bids-specification/compare/c00d1c14d8fe8a2378f1169d5186062017e603c0...6ab3e006fc0286b4bc246cf389ce30d2437050b3

**Comment by JuliusWelzel** [comment]
> @JuliusWelzel Could you review #1591? It contains these proposals:
> 
> [c00d1c1...6ab3e00](https://github.com/bids-standard/bids-specification/compare/c00d1c14d8fe8a2378f1169d5186062017e603c0...6ab3e006fc0286b4bc246cf389ce30d2437050b3)

Looks good to me. Will will update the Examples accordingly. Thanks!

**Comment by JuliusWelzel** [comment]
> Will will update the Examples accordingly.

Examples have been updated [here](https://github.com/bids-standard/bids-examples/pull/418), I think it is ready to be merged.

Minor [changes](https://github.com/bids-standard/bids-specification/pull/1524/commits/98112eba448370f59ef94a6b94d15a0b49067fc7) to specs have been made as well

**Comment by sappelhoff** [comment]
- closing as replaced by https://github.com/bids-standard/bids-specification/pull/1591

**Pull request closed without merging**